docker/coverage

```diff
@@ -28,6 +28,7 @@ docker run --rm --network=$NAME_NETWORK \
        -e VALIDATE_JSON_SCHEMAS=true \
        -e USE_MOCK_MODEL=true \
        -e _R_CHECK_CRAN_INCOMING_=FALSE \
+       -e TESTTHAT_CPUS=4 \
```

Comment:
this should detect the number of cores. Can we add something to our parallel runner to expose this? or detect cpu count on the runners (e.g., lscpu)

Reply:
Nice, see https://github.com/reside-ic/reside-hyperv-scripts/pull/43